core/config/docs/defaults.go

```diff
@@ -0,0 +1,27 @@
+package docs
+
+import (
+	"log"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink/v2/core/config/toml"
+	"github.com/smartcontractkit/chainlink/v2/core/services/chainlink/cfgtest"
+	"github.com/smartcontractkit/chainlink/v2/core/store/dialects"
+	"github.com/smartcontractkit/chainlink/v2/core/utils/config"
+)
+
+var (
+	defaults toml.Core
+)
+
+func CoreDefaults() (c toml.Core) {
+	if (defaults == toml.Core{}) {
```

Comment:
@jmank88 Let me know what you think of this change; instinctively I think it's an anti-pattern to have packages implement init, so I made the instantiation of `defaults` lazy. This shouldn't change any behaviour since CoreDefaults is still one of the very first calls made at startup.

Reply:
How is this an improvement over an init func? A `sync.Once` would be more efficient than a mutex, but with the init you don't need to synchronize at all.